LayoutTests/accessibility/aria-expanded-supported-roles.html

```diff
@@ -92,17 +93,18 @@
 </div>
 
 <script>
-    if (window.accessibilityController) {
-        let output = "Verify which roles support aria-expanded.\n";
+    var output = "Verify which roles support aria-expanded.\n";
 
+    if (window.accessibilityController) {
         Array.from(document.getElementById("content").children).forEach((element) => {
             let axElement = accessibilityController.accessibleElementById(element.id);
             if (axElement && axElement.isExpanded)
                 output += `${axElement.role} is expanded\n`;
         });
 
-        debug(output);
         document.getElementById("content").style.visibility = "hidden";
+        debug(output);
+        finishJSTest();
```

Comment:
`finishJSTest();` is only necessary for tests that have been marked `window.jsTestIsAsync = true;`, and I don't this test is async.